app/frontend/schools/settings/components/SchoolCustomize__AgreementsEditor.res

```diff
@@ -94,6 +99,7 @@ let initialState = (kind, customizations) => {
   let agreement = switch kind {
   | PrivacyPolicy => customizations |> Customizations.privacyPolicy
   | TermsAndConditions => customizations |> Customizations.termsAndConditions
+  | CodeOfConduct => customizations |> Customizations.codeOfConduct
```

Comment:
There are two different CoC editors. Why...?

One is accessed via the _Standings_ page, and another via this _Customization_ page...

I understand that triggering the _Customization_ agreement editor from the _Standings_ page might be awkward, but if a full-page editor exists, why not reuse it by linking to it from the _Customization_ page's footer link?

Reply:
I had kept same type of editor for the `Code of Conduct` edit in customization's page because to make it look simillar to editor of `Terms & Conditions` and `Privacy Policy` agreements